### DISCUSSION TITLE
How to add a new Azure Subscription

### DISCUSSION THREAD

**Discussion by carloscds** [rcl-ssl/docs#5] [General]
How to add a new subscription to create certificates ?

**Comment by rcladmin** [comment]
If you have an existing subscription and want to add a new subscription to the RCL SaaS app, you must login with a different Azure AD Work/School account when you subscribe for a new RCL SaaS subscription in the Azure Portal/Maketplace. 

This is a suggestion how this can be done:

1. Create a new 'User' in your Azure Active directory. The User should NOT be a 'Guest User'.
2. Ensure the new user has 'Contributor' or 'Owner' role to the subscription(s) containing the WebApps /DNS/KeyVault that you want to create the TLS/SSL certificate for
3. Log in with the new user account in the Azure Portal/Marketplace
4. Subscribe to the RCL Client SaaS application in the Azure Portal/Marketplace , ensure you are using the new user account to subscribe

Once the subscription is successful, you must login in with the new user account if the RCL app for the new subscription to create certificates.

**Reply by carloscds** [reply]
I need to create a new RCL SAAS for each subscription ? Why ?

**Reply by rcladmin** [reply]
Ok, now I understand. You need to use the SAME RCL SaaS subscription to cover multiple Azure Subscriptions.

This is how you can accomplish this.

1. Ensure the User account that you use to login to to the RCL SaaS app has 'Owner' or 'Contributor' access to all the Azure Subscriptions that you would need to access.

Your Azure Subscriptions and Azure AAD tenant should be in the same Azure account to accomplish this.

The following link provides more details:

https://docs.rclapp.com/authorization/access-control-user.html

**Reply by carloscds** [reply]
I already add my RCL user as Contributor and Owner of this new subscription, but it does not appear on RCL portal.

**Reply by carloscds** [reply]
If this subscription is in another tenant, it's not possible ?

**Reply by rcladmin** [reply]
Is it possible for you to post a screen shot showing the assigned User to the Azure subscription in the role assignments tab of IAM in the subscription page ? A sample is shown below. If you are concerned about security, you can send me email at support@rclapp.com

<img width="627" alt="Capture" src="https://user-images.githubusercontent.com/4339259/118737069-153a5a80-b812-11eb-978f-4a8940e31a40.PNG">

**Reply by carloscds** [reply]
Solved!
